### PR TITLE
Add documentation for halohome integration

### PR DESCRIPTION
**Pull request by nayaverdier**
## Proposed change

Adds documentation for the new halohome integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/58316
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2909 (previously closed but can be reopened)
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/halohome.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ha_iot_class: Assumed State
+ha_codeowners:
+  - '@nayaverdier'
+ha_domain: halohome
+ha_config_flow: true
+ha_platforms:
+  - light
+---
+
+Integrates [HALO Home](https://www.cooperlighting.com/global/brands/halo-home) smart lights into Home Assistant.
```
Comment (frenck) [inline source/_integrations/halohome.markdown:16]: While correct, it is a little bit uh...
Could be a bit more extensive maybe?

File: source/_integrations/halohome.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Configuration
+
```
Comment (frenck) [inline source/_integrations/halohome.markdown]: This header is already provided by the snippet included below.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/halohome.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+Integrates [HALO Home](https://www.cooperlighting.com/global/brands/halo-home) smart lights into Home Assistant.
+
+Supports brightness and color temperature for Bluetooth HALO Home lights.
+
+Devices are read from the HALO Home API upon setup and stored by this
+integration to allow devices to be controlled even when the internet is not
+accessible. Because of this, if devices are added or removed, the integration
+must be reconfigured.
```
Comment (frenck) [inline source/_integrations/halohome.markdown]: Without looking at the code at this point, that seems weird. A user should not have to reconfigure to add devices. Instead, it should just pick them up when the connection has been restored.

Comment (nayaverdier) [reply source/_integrations/halohome.markdown]: Would it be acceptable to check for new devices when the integration is (re)loaded, or is there another preferred solution?

Comment (frenck) [reply source/_integrations/halohome.markdown]: > Would it be acceptable to check for new devices when the integration is (re)loaded, or is there another preferred solution?

Yeah, that is possible, or maybe poll every now and then to check the state. Having to provide instructions for these things, generally indicate something is a bit off / could be improved for the end-user.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by deadman96385** [comment]
It might be a good idea to put what products are supported because there is quite a large lineup of Halo lights.

**Pull request closed without merging**